ui/app/AppLayouts/Chat/ChatColumn/MessageComponents/NormalMessage.qml

```diff
@@ -122,15 +130,20 @@ Item {
             anchors.topMargin: Style.current.smallPadding
             anchors.left: parent.left
             anchors.leftMargin: chatBox.chatHorizontalPadding
+            onLoaded: {
+                chatImageContent.item.isCurrentUser = isCurrentUser
+            }
 
             sourceComponent: Component {
                 Item {
                     width: chatImageComponent.width + 2 * chatBox.chatHorizontalPadding
                     height: chatImageComponent.height
+                    property bool isCurrentUser
                     ChatImage {
                         id: chatImageComponent
                         imageSource: image
                         imageWidth: 250
+                        isCurrentUser: parent.isCurrentUser
```

Comment:
If the problem is the "circular dependency" or something like that, it's because both have the same name.
You can refer to the "global" `isCurrentUser` with the `id` of the Message parent
```suggestion
                        isCurrentUser: messageItem.isCurrentUser
```

Reply:
This was a left over, removing the loader.item part :)